plugins/woocommerce/includes/admin/marketplace-suggestions/class-wc-marketplace-updater.php

```diff
@@ -43,7 +43,19 @@ public static function update_marketplace_suggestions() {
 
 		$data['updated'] = time();
 
-		$url     = 'https://woocommerce.com/wp-json/wccom/marketplace-suggestions/1.0/suggestions.json';
+		$request_data = array();
+
+		if ( class_exists( 'WC_Marketplace_Suggestions' ) && WC_Marketplace_Suggestions::allow_suggestions() ) {
```

Comment:
with store's location being sent, shouldn't we check whether tracking is allowed too?

```suggestion
		$allow_tracking = 'yes' === get_option( 'woocommerce_allow_tracking', 'no' );
		if ( $allow_tracking && class_exists( 'WC_Marketplace_Suggestions' ) && WC_Marketplace_Suggestions::allow_suggestions() ) {
```

Reply:
That makes sense, yes. Good catch.